mitmproxy/addons/blockech.py

```diff
@@ -0,0 +1,21 @@
+from mitmproxy import ctx
+from mitmproxy import dns
+from mitmproxy.net.dns import types
+
+
+class BlockECH:
+    def load(self, loader):
+        loader.add_option(
+            "blockech",
+            bool,
+            True,
+            "Prevent encrypted SNI by blocking ECH key from passing through by removing HTTPS records from answers",
```

Comment:
Let's spell out acronyms here, not everyone will be familiar with these. How about

> Strip DNS HTTPS records to prevent clients from encrypting TLS ClientHello messages.

? Open to suggestions. :)